tests/readMat.R

```diff
@@ -221,3 +224,21 @@ stopifnot(
   inherits(res, "error"),
   grepl("MAT v7.3 files is not supported", conditionMessage(res))
 )
+
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# Reading ADI LabChart export example file
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+pathname <- file.path(path, "ADI_LabChart_Export.mat")
+
+cat("Reading ADI Labchart file: ", basename(pathname), "\n", sep = "")
+tryCatch({
```

Comment:
One more thing. So, here you don't want to use `tryCatch()`, because you want this test to fail if `readMat()` fails.  So, just do:

```r
mat <- readMat(pathname)
print(mat)
```

You could also add some assertions that the read data is what you'd expect, e.g.

```r
stopifnot(
  length(mat) == 14,
  identical(dim(mat$data), c(1L, 4360L)),
  identical(mat$comtext, character(0))
)
```